src/pages/signup/index.page.tsx

```diff
@@ -0,0 +1,31 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import styles from './Signup.module.scss';
+import SignupForm from '@/pages/signup/components/SignupForm';
+
+function Signup() {
+  return (
+    <div className={styles.container}>
+      <header className={styles.header}>
+        <Image
+          src="/svgs/taskify-vertical-logo.svg"
+          alt="Taskify"
```

Comment:
```suggestion
          alt="Taskify"
```
눈 감고 이 alt 값을 스크린 리더가 읽어주면 어떤 이미지라고 연상하기 어려울 것 같아요 Taskify 로고 이런식으로 적어도 좋을 것 같네요!